plugins/sqlfluff-templater-dbt/setup.cfg

```diff
@@ -62,9 +62,12 @@ packages = find:
 python_requires = >=3.7
 install_requires =
     sqlfluff==1.4.2
-    dbt-core>=0.20.0
+    dbt-core>=1.0.0
     jinja2-simple-tags>=0.3.1
     markupsafe
+    pydantic
+    rich
+    ruamel.yaml
```

Comment:
I assume these are dependencies of `dbt-osmosis`. `rich` looks interesting - should we also pull that one up to our main package and replace `colorama`?

Reply:
We might consider it. IIUC, it's a much bigger, more ambitious project, almost more like a UI framework. By comparison, `colorama` is a pretty simple way to add color to text output.

Reply:
understood.